### PR TITLE
[Qt] Update to 6.7.2

### LINKED ISSUE #39377: [qt] update to 6.7.2
### Library name

qt

### New version number

6.7.2

### Other information that may be useful (release notes, etc...)

https://code.qt.io/cgit/qt/qtreleasenotes.git/about/qt/6.7.2/release-note.md

### PR DESCRIPTION
**Pull request by Neumann-A**
closes #39377

### CODE REVIEW

File: ports/qtmultimedia/remove-static-ssl-stub.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
++++ b/src/plugins/multimedia/ffmpeg/cmake/QtAddFFmpegStubs.cmake
+@@ -70,7 +70,10 @@ macro(qt_internal_multimedia_find_openssl_soversion)
+         if (NOT OPENSSL_SSL_LIBRARY)
+             message(FATAL_ERROR "OPENSSL_SSL_LIBRARY is not found")
+         endif()
+-
++        set(openssl_is_shared ON)
++        if(OPENSSL_SSL_LIBRARY MATCHES "${CMAKE_STATIC_LIBRARY_SUFFIX}$")
++          set(openssl_is_shared OFF)
++        endif()
```
Comment (dg0yt) [inline ports/qtmultimedia/remove-static-ssl-stub.patch:13]: This won't detect shared for windows (`.lib`) and mingw (`.dll.a`) import libs.

Comment (Neumann-A) [reply ports/qtmultimedia/remove-static-ssl-stub.patch:13]: Yeah I know. I don't think it matters:
https://github.com/qt/qtmultimedia/blob/532e685bc80eb17b4d3c397ad5b34fefdebd98a1/src/plugins/multimedia/ffmpeg/cmake/QtAddFFmpegStubs.cmake#L69

I don't know why the qtmultimedia team seems to add unnecessary stubs just to be able to locate dependencies when deploying. So many wrong assumptions here.

**Comment by jimwang118** [comment]
Add reviewed tag for further review by the vcpkg team.

File: ports/qtbase/cmake/qt_install_submodule.cmake [hunk 2]
```diff
@@ -26,7 +26,7 @@ function(qt_download_submodule_impl)
         if(PORT STREQUAL "qttools") # Keep this for beta & rc's
             vcpkg_from_git(
                 OUT_SOURCE_PATH SOURCE_PATH_QLITEHTML
-                URL git://code.qt.io/playground/qlitehtml.git # git://code.qt.io/playground/qlitehtml.git
+                URL https://code.qt.io/playground/qlitehtml.git
```
Comment (BillyONeal) [inline ports/qtbase/cmake/qt_install_submodule.cmake:29]: Thanks!

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**